tests/plugins/kernel_install/utils_test.sh

```diff
@@ -41,8 +41,6 @@ function tearDown()
 
 function test_cmd_manager()
 {
-  local count=0
```

Comment:
I believe a recent commit to this file removed the other two. When I did the rebase, it had a conflict on it. Should we change the commit message or just take this commit out since there is only one variable being removed now? I already thought that this was too small to be a commit, but now even more.

Reply:
I think you can remove this commit from this PR and create a separate PR with this change but also see if you can find some other simple updates to this file, like replacing double quotes with single quotes where appropriate.

Reply:
Ok, I believe is for the best too. We will do this way